task_DEV-2/task_DEV-2/StringWork.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task_DEV_2
+{
+    class StringWork
+    {
+
+
+        private string[] EnglishTranslit = {"A", "B", "V", "G","D","E", "YO", "ZH", "Z",
+                "I", "Y", "K", "L", "M", "N", "O", "P", "R", "S", "T", "U", "F", "KH",
+                "TS", "CH", "SH", "SCH", "", "Y", "", "E", "YU", "YA"  };
+
+         private string[] EnglishToRussianENG = { "SCH", "YO", "ZH", "KH", "TS", "CH", "SH", "YU", "YA", "Y",
+            "E", "A", "B", "V", "G", "D", "Z", "I", "K", "L", "M", "N", "O", "P" , "R", "S", "T", "U", "F", "J", "C", "H", "Q", "X"};
+
+        string[] EnglishToRussianRUS = {  "(Щ/СЧ)", "Ё", "Ж", "Х", "Ц", "Ч", "Ш", "Ю", "Я", "(Й/Ы)",
+            "(Е/Э)", "А", "Б", "В", "Г", "Д", "З", "И", "К", "Л", "М", "Н", "О", "П" , "Р", "С", "Т", "У", "Ф", "ДЖ", "СИ", "Х", "КУ", "ИКС" };
+
+        public bool IsRussian(string s)
+        {
+            s = s.ToUpper();
+            for (int i = 0; i < s.Length; i++)
+            {
+                int ValueOf = (int)s[i];
```

Comment:
bug, it will return true after the first russian symbol and not check others. What is 1040 and 1071? Which code table? magics